docs/HowToSupportProtectedBuild.md

```diff
@@ -4,11 +4,14 @@
 - [Overview of Flat Build](#overview-of-flat-build)
 - [Overview of Protected Build](#overview-of-protected-build)
 - [Steps for supporting protected build in a new chipset](#steps-for-supporting-protected-build-in-a-new-chipset)
-	- [Make.defs](#makedefs)
+	- [Prerequisites](#prerequisites)
 	- [Linker scripts](#linker-scripts)
 	- [Userspace Initialization](#userspace-initialization)
 	- [MPU Initialization](#mpu-initialization)
 	- [Allocation of kernel and user heaps](#allocation-of-kernel-and-user-heaps)
+	- [Make file changes](#make-file-changes)
```

Comment:
I think we'd better use `Makefile` instead of `Make file`.

Reply:
updated